internal/pkg/agent/install/flavors.go

```diff
@@ -48,7 +48,7 @@ func UsedFlavor(topPath, defaultFlavor string) (string, error) {
 		return "", fmt.Errorf("failed reading flavor marker file: %w", err)
 	}
 
-	return string(content), nil
+	return strings.TrimSpace(string(content)), nil
```

Comment:
What bug is this fixing? It's not clear there's a test for it.

Reply:
i've seen weird formatting in some test, i do not recall which one